Containerfile.bpfman-agent

```diff
@@ -37,12 +37,12 @@ RUN echo "TARGETOS=${TARGETOS}  TARGETARCH=${TARGETARCH}  BUILDPLATFORM=${BUILDP
 
 WORKDIR /usr/src/cri-tools
 ARG CRI_REPO_URL=https://github.com/kubernetes-sigs/cri-tools
-ARG CRI_REPO_TAG=v1.32.0
+ARG CRI_REPO_BRANCH=master
```

Comment:
can u pls see if they have released new version so we can use it as new pinned tag instead of master ?